### PR TITLE
Feat: SpringCloud subscribe strategy and Adapter add enabled config

### PR DESCRIPTION
**Pull request by PhilYue**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #
#410 

**Special notes for your reviewer**:
- SpringCloud firstFetch by subscribe strategy
- User control Adapter is or not enable
**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by mark4z** [comment]
Good job

**Review by AlexStocks — APPROVED** [review]

File: pkg/model/adapter.go [hunk 1]
```diff
@@ -19,7 +19,8 @@ package model
 
 // Adapter the adapter plugin for manage cluster or router
 type Adapter struct {
-	ID     string                 `yaml:"id" json:"id"`
-	Name   string                 `yaml:"name" json:"name"`                           // Name the adapter unique name
-	Config map[string]interface{} `yaml:"config" json:"config" mapstructure:"config"` // Config adapter config
+	ID      string                 `yaml:"id" json:"id"`
+	Name    string                 `yaml:"name" json:"name"` // Name the adapter unique name
+	Enabled string                 `yaml:"enabled" json:"enabled" default:"true"`
```
Comment (mark4z) [inline pkg/model/adapter.go:24]: 'Enabled' why not bool?

**Review by mark4z — APPROVED** [review]

File: pkg/adapter/springcloud/cloud.go [hunk 2]
```diff
@@ -347,8 +364,15 @@ func (a *CloudAdapter) stop() error {
...
 	}
 	close(a.stopChan)
 	return nil
 }
+
+func (a *CloudAdapter) subscribeServiceDefinitely() bool {
+	return strings.EqualFold(a.cfg.SubscribePolicy, Definitely.String())
+}
+
+func (a *CloudAdapter) subscribeServiceAdapting() bool {
```
Comment (ztelur) [inline pkg/adapter/springcloud/cloud.go]: pl fix review dog comment issues

File: pkg/model/adapter.go (hunk 1 above)
Comment (PhilYue) [reply pkg/model/adapter.go:24]: 启动时，加载配置时 默认值函数（default.Set）会吧自定义的值覆盖了

**Review by ztelur — APPROVED** [review]

**Pull request merged**